### PR TITLE
Extract ReportPortal Launch URL to File.

### PR DESCRIPTION
**Pull request by ekorneeva24**
https://github.com/hulkag/cch-platform/pull/1027

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ const RP_DEBUG_MODE = 'DEBUG';
 const RP_DEFAULT_MODE = 'DEFAULT'
 
 const LAUCH_ID_FILE_NAME = 'LAUNCH_ID';
+const LAUCH_URL_FILE_NAME = 'LAUNCH_URL'
```
Comment (dwentland24) [inline index.js]:
```suggestion
const LAUNCH_URL_FILE_NAME = 'LAUNCH_URL'
```

File: index.js [hunk 2]
```diff
@@ -359,6 +360,12 @@ module.exports = (config) => {
 
       reportUrl = response.link;
       output.print(` 📋 Report #${response.number} saved ➡`, response.link);
+
+      fs.writeFile(LAUCH_URL_FILE_NAME, response.link, function (err) {
```
Comment (dwentland24) [inline index.js]:
```suggestion
      fs.writeFile(LAUNCH_URL_FILE_NAME, response.link, function (err) {
```

File: index.js [hunk 3]
```diff
@@ -359,6 +360,12 @@ module.exports = (config) => {
...
+        if (err) return console.log(err);
+        console.log("Output Lauch Url to a file " + LAUCH_URL_FILE_NAME);
```
Comment (dwentland24) [inline index.js]:
```suggestion
        output.print(`Output Launch Url to file: ${LAUNCH_URL_FILE_NAME}`);
```

**Review by dwentland24 — COMMENTED** [review]
@ekorneeva24 Made some suggestions regarding a typo and  the log output.

**Comment by ekorneeva24** [comment]
@dwentland24 thanks, applied your changes and also fixes the similar typo in LAUCH_ID_FILE_NAME -> LAUNCH_ID_FILE_NAME

**Review by dwentland24 — APPROVED** [review]

**Pull request merged**